recipes/qpot/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "qpot" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "0ecb77f5e4cab70a49946f9ecc3e32640e64ab4b0e148c9ccd67cf46006174db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/QPot/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - xtensor
+  run:
+    - xtensor
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/QPot  # [unix]
+    - test -f ${PREFIX}/include/QPot/Static.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/QPot/QPotConfig.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\QPot\Static.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\QPotConfig.cmake (exit 0) else (exit 1)  # [win]
```

Comment:
Can you add tests to ensure that shared libraries are being installed and not static ones?

Reply:
It is header only, there are no shared libraries.